tests/test_env.py

```diff
@@ -116,10 +116,20 @@ def test_req_parser(tmpdir):
 
 def test_gen_req_file(tmpdir):
     e = env.VirtualEnv(tmpdir)
-    req = ["lorem == 0.1.1", "lorem > 0.1", "dummy-yummy", "iplib@git+https://github.com/bartv/python3-iplib", "lorem"]
+    req = [
+        "lorem == 0.1.1",
+        "lorem > 0.1",
+        "dummy-yummy",
+        "iplib@git+https://github.com/bartv/python3-iplib",
+        "lorem",
+        # verify support for environment markers as described in PEP 508
+        "lorem;python_version<'3.7'",
```

Comment:
I would also add a test that verifies the behavior when a requirements contains two markers on the same line.